src/netcdf_writer_coordinates.jl

```diff
@@ -112,11 +115,16 @@ function target_coordinates(space, num_points) end
 
 function target_coordinates(
     space::S,
-    num_points,
+    num_points;
+    disable_vertical_interpolation,
 ) where {
     S <:
     Union{Spaces.CenterFiniteDifferenceSpace, Spaces.FaceFiniteDifferenceSpace},
 }
+    if disable_vertical_interpolation
+        return Array(parent(space.grid.center_local_geometry.coordinates))[:, 1]
```

Comment:
```suggestion
        return Array(parent(Fields.coordinate_field(space).z)[:, 1]
```
I think we'd better use a ClimaCore getter here instead, is this basically `Fields.coordinate_field`?

Reply:
Actually, `vec(parent(Fields.coordinate_field(space).z[Fields.ColumnIndex((1,1), 1)]))` might be even more general (as it does not rely on the DataLayout's array), but I'm not sure if this (or the existing implementation) is valid with topography